jsonlogging/src/main/java/org/secondbase/logging/JsonLogger.java

```diff
@@ -0,0 +1,215 @@
+     */
+    public static void setupLoggingStdoutOnly(
+            final String environment,
+            final String service,
+            final String datacenter,
+            final String requestLoggerName) {
+        setupLoggingStdoutOnly(environment, service, datacenter, requestLoggerName, true);
+    }
+
+    /**
+     * Set up logback logging for service and request logs. We use only ONE appender here to write
+     * normal log lines to stdout for both types. However, the request logger is set up slightly
+     * differently. To guarantee that everything works as expected this method must be called with
+     * the fully qualified class name of the request logger that will be used in jetty.
+     * @param environment field to add to all log lines
+     * @param service field to add to all log lines
+     * @param datacenter field to add to all log lines
+     * @param requestLoggerName the request logger class name
+     * @param json true for json output, false for plain test
+     */
+    public static void setupLoggingStdoutOnly(
+            final String environment,
+            final String service,
+            final String datacenter,
+            final String requestLoggerName,
+            final boolean json) {
+        final LoggerContext loggerContext = getLoggerContext(environment, service, datacenter);
+        final Appender<ILoggingEvent> consoleAppender = json
+                ? createJsonConsoleAppender(SERVICE_CONSOLE_APPENDER, loggerContext, true)
+                : createPatternLayoutConsoleAppender(SERVICE_CONSOLE_APPENDER, loggerContext, true);
+
+        // specifically cast to logback version so we set it up
+        final Logger rootLogger = (Logger) LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
+        rootLogger.addAppender(consoleAppender);
+        rootLogger.setLevel(LOG_LEVEL);
+
+        if (!Strings.isNullOrEmpty(requestLoggerName)) {
+            final Logger requestLogger = (Logger) LoggerFactory.getLogger(requestLoggerName);
+            requestLogger
+                    .addAppender(json
+                            ? createJsonConsoleAppender(
+                                    REQUESTLOG_CONSOLE_APPENDER, loggerContext, false)
+                            : createPatternLayoutConsoleAppender(
+                                    REQUESTLOG_CONSOLE_APPENDER, loggerContext, false));
+            requestLogger.setAdditive(false);
+            requestLogger.setLevel(LOG_LEVEL);
+        }
+    }
+
+    static LoggerContext getLoggerContext(
+            final String environment,
+            final String service,
+            final String datacenter) {
+        // specifically cast to logback version so we can add advanced stuff (i.e. properties)
+        final LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+        loggerContext.reset();
+
+        if (Strings.isNullOrEmpty(service)) {
```

Comment:
This is always true, since it has a default value. Unless someone sets it specifically to zero. I think it should not have default values, so that it fails when additional options are needed.